docs/Customization/Memory-Hierarchy.rst

```diff
@@ -16,6 +16,12 @@ configure 4 KiB direct-mapped caches for L1I and L1D.
 
     import freechips.rocketchip.subsystem.{WithNMediumCores, WithNSmallCores}
 
+    class SmallRocketConfig extends Config(
+    new WithTop ++                                       // use default top
```

Comment:
Can we change this to be indented like the other configs?

Reply:
Done!